schema/schema_changelog.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "acceptsSchemaHash": "d8fc391a26a0c08d334ad6380a41534f",
+    "acceptsSchemaHash": "65f8199100ba2cbd44209f81249d8599",
```

Comment:
Required because I changed a default value (@oetherington we're planning to not put default values in the schema eventually right?)